packages/app/src/components/Sliders/Slider.module.css

```diff
@@ -22,49 +27,21 @@
   }
 }
 
-.value {
-  justify-self: end;
-}
-
-.slider {
-  appearance: none;
-  background: transparent;
-  outline: none;
+.slider-wrapper {
+  display: block;
```

Comment:
this declaration does nothing, because `div` is a block element by default. Please make sure there are no unused declarations in this change. I can't tell just by reading it. For example, all the `position: absolute`, `width: 100%`, `height: 100%` are they necessary? Would it be solvable using `flex`?